src/UI/Utils/useTicker.test.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { act, render } from "@testing-library/react";
+import { useTickerWithInterval, useTickerWithTimestamp } from "./useTicker";
+
+jest.useFakeTimers();
+
+test("GIVEN useTickerWithInterval WHEN provided with interval THEN executes at that interval", () => {
+  const callback = jest.fn();
+  const Component: React.FC = () => {
+    useTickerWithInterval("OneSecond");
+    callback();
+    return null;
+  };
+  render(<Component />);
+
+  act(() => {
+    jest.advanceTimersByTime(1000);
+  });
+
+  expect(callback).toBeCalledTimes(2);
+});
+
+test("GIVEN useTickerWithTimestamp WHEN provided with timestamp longer than 2 hours ago THEN never executes again", () => {
```

Comment:
I'm not sure about this use case.
If we set a limit like this, and someone leaves a tab open overnight, it will still show `2 hours ago` (and suddenly updates when the user navigates) which is not great, and this can also happen if we increase the limit.
If we don't have a limit, then it could potentially run forever, which feels a bit like a waste of resources.

We can go ahead with this, and still change it later based on feedback.